### DISCUSSION TITLE
Tomcat SignleSignOn valve implementation for Redis

### DISCUSSION THREAD

**Discussion by iamrakesh** [redisson/redisson#5657] [General]
As a follow up to https://github.com/redisson/redisson/issues/5603, below is my attempt to implement SignleSignOn valve to save SSO entries to Redis.
https://github.com/redisson/redisson/compare/master...iamrakesh:redisson:master

Could any of the maintainers do a quick review and guide me with next steps.
Thank you

**Comment by mrniko** [comment]
Thank you for the proposed changes. 

is it possible to replace redissonClient.getMap with RedissonSessionManager.getMap() method ? This method is aware of `keyPrefix` setting.
